src/components/login-view/login-veiw.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+
+import "./login-view.scss";
+
+export function LoginView(props) {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(username, password);
+    // Send a request to the server for authentication then call props.onLoggedIn(username)
+    props.onLoggedIn(username);
+  };
+
+  return (
+    <div className="login-view">
+      <Form className="login-form">
+        <Form.Group controlId="formUsername">
+          <Form.Label>Username:</Form.Label>
+          <Form.Control
+            type="text"
+            value={username}
+            onChange={(e) => setUsername(e.target.value)}
+            placeholder="Enter Username"
+          />
+        </Form.Group>
+
+        <Form.Group controlId="formBasicPassword">
+          <Form.Label>Enter Password</Form.Label>
+          <Form.Control
+            type="password"
+            value={password}
+            onChange={(e) => setPassword(e.target.value)}
+            placeholder="Password"
+          />
+        </Form.Group>
+        <Button variant="secondary" type="button" onClick={handleSubmit}>
+          Login
+        </Button>
+      </Form>
```

Comment:
Add another button here for `Register` and use the newly created function to show the registration page.

```
  <Button variant="secondary" type="button" onClick={this.props.onRegister}>
          Register
  </Button>
```